.changeset/bright-radios-kiss.md

```diff
@@ -0,0 +1,6 @@
+---
+"kilo-code": patch
+"@roo-code/vscode-webview": patch
```

Comment:
@kevinvandijk now that there are a lot of new package (names) incoming, we might have to change this too.

Not actionable on your part @HadesArchitect